docs/developer/developing.md

```diff
@@ -0,0 +1,49 @@
+# Developing Cincinnati
+
+## Overview
+
+The following sections describe how to build and test the project for local development. 
+
+
+## Prerequisites
+
+* Git
+* Rust
+* [just](https://github.com/casey/just)
+
+*Note:* We recommend using [Rustup](https://github.com/rust-lang/rustup/blob/master/README.md) for Rust toolchain management. 
+
+## Running it locally
+
+Please below commands in different terminal windows 
+```
+$ just run-graph-bulider
+```
+
+```
+$ just run-policy-engine
+```
+
+Here is the command to get graph for "stable-4.2" and for "amd64" architecture
+
+```
+just get-graph-pe "stable-4.2" "amd64"
+```
+
+## Building it locally
+
+TBD
```

Comment:
What exactly do you mean by *it*? And do you mean just to build the binaries? Or a container image?

Reply:
Both. Also planning to add the steps where we can build the image locally and push it to quay.io and then use it to run on an OpenShift instance.